Riot/Modules/Rendezvous/RendezvousTransport.swift

```diff
@@ -33,12 +34,14 @@ class RendezvousTransport: RendezvousTransportProtocol {
     }
     
     func get() async -> Result<Data, RendezvousTransportError> {
-        guard let url = rendezvousURL else {
-            return .failure(.rendezvousURLInvalid)
-        }
-        
         // Keep trying until resource changed
         while true {
```

Comment:
We rely on the backend to expire it for us so even if the teardown fails for some reason this will still get cleaned up eventually. Here's the same thing on the js-sdk https://github.com/matrix-org/matrix-js-sdk/blob/hughns/http-rendezvous/src/rendezvous/transports/simpleHttpTransport.ts#L146